src/watermark.plugin.js

```diff
@@ -0,0 +1,60 @@
+function WikiaJWPlayerWatermarkPlugin(player, config, div) {
+	this.player = player;
+	this.container = div;
+	this.config = config;
+	this.watermarkElement = this.getWatermarkElement();
+	this.watermarkElement.addEventListener('click', function () {
+		player.trigger('watermarkClicked');
+	});
+
+	this.container.classList.add('wikia-watermark-container');
+
+	this.isEnabled = config.show;
+
+	this.player.on('play', this.show.bind(this));
+	this.player.on('pause', this.hide.bind(this));
+	this.player.on('relatedVideoPlay', this.onVideoChange.bind(this));
+}
+
+/**
+ * prepares watermark link with icon
+ */
+WikiaJWPlayerWatermarkPlugin.prototype.getWatermarkElement = function () {
+	var watermarkImage = wikiaJWPlayerIcons['fandomLogo'];
+
+	var watermarkElement = document.createElement('a');
+	watermarkElement.classList.add('wikia-watermark');
+	watermarkElement.innerHTML = watermarkImage;
+	watermarkElement.href = 'https://fandom.com';
```

Comment:
probably not for now, but we could parametrise this link somehow, so video team could specify where specific wordmark redirects.

Reply:
Yeah, it will be easy to do so, once we have more than one watermark :)